onnxscript/ir/_external_data_test.py

```diff
@@ -1,12 +1,16 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
+import pathlib
+import tempfile
 import unittest
 
 import onnx
 import onnx.external_data_helper
 
+import numpy as np
+
 from onnxscript import ir
-from onnxscript.ir import _external_data
+from onnxscript.ir import _core, _external_data
```

Comment:
```suggestion
from onnxscript.ir import _external_data
```

for the classes that is not from _external_data (aka not tested in this file) use `ir.xxx` directly so when users are using our tests as reference they can write the same thing and not have to import internal modules.